library/core/src/slice/mod.rs

```diff
@@ -2035,7 +2035,12 @@ impl<T> [T] {
         //
         // `[ptr; mid]` and `[mid; len]` are not overlapping, so returning a mutable reference
         // is fine.
-        unsafe { (from_raw_parts_mut(ptr, mid), from_raw_parts_mut(ptr.add(mid), len - mid)) }
+        unsafe {
```

Comment:
two local variables for the left and right sides would have been nice here to reduce the lines from 6 to 3 :3, things like this are certainly formative of your opinions about rustfmt :laughing: (though i can't really complain, it's doing its best)